xbmc/music/windows/GUIWindowMusicNav.cpp

```diff
@@ -287,6 +287,10 @@ bool CGUIWindowMusicNav::GetDirectory(const std::string &strDirectory, CFileItem
       OnRetrieveMusicInfo(items);
   }
 
+  //Navigating music files so default content to "songs" unless in sources folder.
+  //This content allows view type to include media info so that file tag data can be displayed
+  if (!URIUtils::IsSourcesPath(strDirectory))
```

Comment:
>Have you ever been to the video files views?
Yes, most evenings. But I don't have a video library of scraped info so I don't see what you mention. 

Music is different in that there is intrinsic data to display, part of the file as much as date or size, not additional data scraped from external sources.

But clearly my implementation is faulty and is causing problems. Sorry about that, I will look to fixing it. Suggestions welcome.